### PR TITLE
chore(qr): Move QR to profile

### PR DESCRIPTION
**Pull request by legitatx**
Moved QR code to user profile

### CODE REVIEW

**Review by MichaelZhao21 — APPROVED** [review]
Remove the QR code from the `/dashboard/scan-in` and honestly just remove that page

**Review by MichaelZhao21 — CHANGES_REQUESTED** [review]
See my comment above

**Comment by legitatx** [comment]
> Remove the QR code from the `/dashboard/scan-in` and honestly just remove that page

Just removed that page in latest commit

File: pages/dashboard/scan-in.tsx [hunk 1]
```diff
@@ -1,92 +0,0 @@
...
-  const [qrLoading, setQRLoading] = useState(false);
-  const [error, setError] = useState('');
-  const [userData, setUserData] = useState('');
-
-  const fetchQR = () => {
-    if (!isSignedIn) return;
-    setQRLoading(true);
-    const query = new URL(`http://localhost:3000/api/applications/${user.id}`);
-    query.searchParams.append('id', user.id);
-    fetch(query.toString().replaceAll('http://localhost:3000', ''), {
```
[the remark this answers is not in the document]
Comment (NickBurnett) [reply pages/dashboard/scan-in.tsx:28]: @abdullah50053 All I'll say is I did exactly what I was supposed to for the system to work for HackUTD. Proof below checkmate.

https://github.com/acmutd/hackportal/blob/556d442ac0e6ca275f43d81fd40e595e9680489e/pages/api/applications/index.tsx#L101
https://github.com/acmutd/hackutd-viii-site/blob/ef4fbea74de9573cc367461a5074e9e11ac16b48/pages/api/applications/index.tsx#L99

[the remark this answers is not in the document]
Comment (MichaelZhao21) [reply pages/dashboard/scan-in.tsx:28]: CAN WE JUST MERGE THIS

**Review by MichaelZhao21 — APPROVED** [review]
good shit

File: pages/dashboard/scan-in.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (adityarathod) [reply pages/dashboard/scan-in.tsx:28]: 🍿 🍿

**Review by NickBurnett — APPROVED** [review]
💯

**Pull request merged**